README.md

```diff
@@ -82,26 +82,46 @@ vars:
+### Passing Through Additional Metrics
+By default, this package will select `clicks`, `impressions`, `cost` and `conversion_value_in_local_currency` (as well as fields set via `linkedin_ads__conversion_fields` in the next section) from the source reporting tables `ad_analytics_by_campaign` and `ad_analytics_by_creative` to store into the corresponding staging models. If you would like to pass through additional metrics to the staging models, add the below configurations to your `dbt_project.yml` file. These variables allow for the pass-through fields to be aliased (`alias`) and transformed (`transform_sql`) if desired, but not required. Only the `name` of each metric field is required. Use the below format for declaring the respective pass-through variables:
 
 ```yml
 # dbt_project.yml
 vars:
     linkedin_ads__campaign_passthrough_metrics: # pulls from ad_analytics_by_campaign
         - name: "new_custom_field"
-          alias: "custom_field"
+          alias: "custom_field_alias"
+          transform_sql: "coalesce(custom_field_alias, 0)" # reference the `alias` here if you are using one
         - name: "unique_int_field"
           alias: "field_id"
+        - name: "another_one"
+          transform_sql: "coalesce(another_one, 0)" # reference the `name` here if you're not using an alias
         - name: "that_field"
     linkedin_ads__creative_passthrough_metrics: # pulls from ad_analytics_by_creative
         - name: "new_custom_field"
           alias: "custom_field"
         - name: "unique_int_field"
 ```
 
+>**Note** Please ensure you exercised due diligence when adding metrics to these models. The metrics added by default (clicks, impressions, and spend) have been vetted by the Fivetran team maintaining this package for accuracy. There are metrics included within the source reports, for example metric averages, which may be inaccurately represented at the grain for reports created in this package. You will want to ensure whichever metrics you pass through are indeed appropriate to aggregate at the respective reporting levels provided in this package. (**Important**: You do not need to add conversions in this way. See the following section for an alternative implementation.)
+
+### Adding in Conversion Fields Variable
+Separate from the above passthrough metrics, the package will also include conversion metrics based on the `linkedin_ads__conversion_fields` variable, in addition to the `conversion_value_in_local_currency` field.
+
+By default, the data models consider `external_website_conversions` and `one_click_leads` to be conversions. These should cover most use cases, but, say, if you would like to consider landing page clicks and external post click conversions to *also* be conversions, you would apply the following configuration:
+
+```yml
+# dbt_project.yml
+vars:
+    linkedin_ads__conversion_fields: ['external_website_conversions',  'one_click_leads', 'external_website_post_click_conversions', 'landing_page_clicks']
```

Comment:
I did some tests and found that this variable **NEEDS** to be the name of the field as it originally is from the source. We should clarify to customers that the field input into this variable will be the `name` and not the `alias` if there is one from the above passthrough fields.